packages/contracts/src/ubiquistick/SimpleBond.sol

```diff
@@ -135,10 +135,13 @@ contract SimpleBond is ISimpleBond, Ownable, Pausable {
         // @dev must set token allowance for this smart contract previously
         IERC20(token).safeTransferFrom(msg.sender, address(this), amount);
 
-        Bond memory bnd;
-        bnd.token = token;
-        bnd.amount = amount;
-        bnd.block = block.number;
+        Bond memory bnd = Bond({
```

Comment:
nice change, can you rename the state variable to something clearer, readable, that does not clash with the `Bond struct`